assets/sass/components/_topbar.scss

```diff
@@ -0,0 +1,141 @@
+.topbar {
+	background-color: $aplus-brand-primary;
+	border-bottom: none;
+	-webkit-box-shadow: 0 1px 4px rgba(0, 0, 0, 0.25),
+		inset 0 -1px 0 rgba(0, 0, 0, 0.1);
+	-moz-box-shadow: 0 1px 4px rgba(0, 0, 0, 0.25),
+		inset 0 -1px 0 rgba(0, 0, 0, 0.1);
+	box-shadow: 0 1px 4px rgba(0, 0, 0, 0.25), inset 0 -1px 0 rgba(0, 0, 0, 0.1);
+
+	& .navbar-brand,
+	& .navbar-nav > li > a,
+	& .navbar-toggle,
+	& .navbar-nav > .active > a,
+	& .navbar-nav > .active > a:hover {
+		@include focus-with-background-and-color;
+
+		color: #ffffff;
+		background: transparent;
+	}
+
+	& a.navbar-brand:hover,
+	& .navbar-nav > li > a:hover,
+	& .navbar-nav > .open > a,
+	& .navbar-nav > .open > a:focus,
+	& .navbar-nav > .open > a:hover,
+	.navbar-inverse .navbar-toggle:focus,
```

Comment:
The old CSS has the rule `.navbar-inverse .navbar-toggle:focus` without nesting under `.topbar`. However, if I understand SASS correctly, now this rule is nested under `.topbar` even if the line does not start with `& `. That is semantically different than the original code, though maybe it does not affect the outcome.

Reply:
Yes, you are right. it was a typo.